core/login.go

```diff
@@ -17,15 +17,18 @@ package core
 
 import (
 	"fmt"
-	"net/url"
 	. "github.com/vmware/priam/util"
+	"net/url"
 )
 
+/* Local user domain as defined in IDM (might change in the future when multiple local domains will be supported) . */
+const LOCAL_USER_DOMAIN = "Local Users"
```

Comment:
The problem is that IDM expects this to be named like that, so it's a constant for now